File: pascal/control_flow/continue_instruction/continue_instruction.pas
(*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 *)

program ContinueInstruction;

var
  i: integer = 0;

begin
  while i < 10 do
  begin
    writeln(i, '...');
    i := i + 1;

    if (i > 5) then
      continue;

    writeln('*********************');
  end;
end.

File: java/control_flow/continue_instruction/ContinueInstruction.java
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

class ContinueInstruction {
  public static void main(String[] args) {
    int i = 0;

    while (i < 10) {
      System.out.println(i + "...");
      i++;

      if (i > 5)
        continue;

      System.out.println("*********************");
    }
  }
}
